allen-cell-animated/simularium-website
# Feature/use full AICS logo in nav bar

Problem
=======
With the launch of the new allencell.org, we want AICS web apps (Simularium, CFE, etc.) to use a top nav that looks like allencell.org's top nav. Simularium currently displays a no-text AICS logo on the top left of the web site. This is inconsistent with the full logo on the allencell.org megamenu.

Resolves: [Replace AICS logo in nav bar with full logo](https://aicsjira.corp.alleninstitute.org/browse/AGENTVIZ-1423)

Solution
========
* Replaced abbreviated logo with full logo
* Restructured the `AppHeader` container to accommodate the wider logo image and to center the title (name of the trajectory in view) better -- that's been bugging me (and probably others) for a while

## Type of change
Please delete options that are not relevant.


Change summary / key files:
---------------
* src/components/Icons/index.tsx - swap logo file
* src/containers/AppHeader/index.tsx - Instead of using the Antd component `PageHeader`, I'm now just using a regular `div` for easier styling
* src/containers/AppHeader/style.css
    * Remove all old styling having to do with overriding the default styling for the Antd `PageHeader` component
    * Group all the media query selectors together at the bottom
    * With the new wider AICS logo, not everything fits in a smartphone portrait view anymore, so hide the "SIMULARIUM HOME" link when the screen is that small
    * Other necessary styling adjustments

Steps to Verify:
----------------
1. Pull this branch
2. Check that the logo has been replaced
3. Load any trajectory
3. Check that the viewer title (trajectory name) is better centered than before
4. Check that the small screen warning still pops up when it's supposed to (< 900 px) and the nav bar generally behaves/looks the same way it used to

Screenshots (optional):
-----------------------
![image](https://user-images.githubusercontent.com/12690133/118893553-72d4b280-b8b7-11eb-9500-aac2ea520709.png)
![image](https://user-images.githubusercontent.com/12690133/118893578-7bc58400-b8b7-11eb-8477-3ba979dd296f.png)
![image](https://user-images.githubusercontent.com/12690133/118893672-add6e600-b8b7-11eb-88e7-56524d7e6b1a.png)
